packages/frameworks/src/frameworks.ts

```diff
@@ -650,7 +617,6 @@ export const frameworks = [
       },
     },
     dependency: '@dojo/cli',
-    getFsOutputDir: async () => 'output/dist',
     getOutputDirName: async () => join('output', 'dist'),
```

Comment:
Unrelated, but why is this one using `path.join()`?

Reply:
Probably so it could work on Windows if we ever ran this function in `vc dev` for example

Reply:
Shouldn't really make a difference I think. Windows file paths support backslashes.